### PR TITLE
Add note about new actions to example

### PR DESCRIPTION
**Pull request by DIRECTcut**
Mention in the 'scroll' example that TestCafe now has  `scroll` actions

### CODE REVIEW

**Review by miherlosev — APPROVED** [review]

File: examples/scroll/README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 **Test Code**: [index.js](index.js)  
 
+> UPDATE: TestCafe v1.14 release adds scroll actions to the test controller. There is no need to use client functions or `hover` to scroll the webpage. Read the documentation for details:
```
Comment (titerman) [inline examples/scroll/README.md:5]: 'version' and 'release' are synonymous

```suggestion
> UPDATE: TestCafe v1.14 includes dedicated scroll actions. You do not need to interact with off-screen elements, or execute client-side code to scroll the webpage. Read the documentation for details:
```

**Review by titerman — CHANGES_REQUESTED** [review]

**Review by titerman — APPROVED** [review]

File: examples/scroll/README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 **Test Code**: [index.js](index.js)  
 
+> UPDATE: TestCafe v1.14 includes dedicated scroll actions. You do not need to interact with off-screen elements, or execute client-side code to scroll the webpage. Read the documentation for details:
```
Comment (arminal) [inline examples/scroll/README.md:5]:
```suggestion
> UPDATE: TestCafe v1.14 includes dedicated scroll actions. You do not need to interact with off-screen elements or execute client-side code to scroll the webpage. Read the documentation for details:
```

File: examples/scroll/README.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
...
+>
+> * [t.scroll](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/scroll.html)
+> * [t.scrollBy](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/scrollby.html)
+> * [t.scrollIntoView](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/scrollintoview.html)
+
 TestCafe [actions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html) have a built-in scrolling mechanism. In rare cases, you may need to scroll the page manually. This example demonstrates two common ways to do this: a `hover` action and a [ClientFunction](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html)
 
 The test page has two buttons. They are located at 2000 px and 4000 px along the X-axis, respectively (both outside of the viewport).
```
Comment (arminal) [inline examples/scroll/README.md:13]:
```suggestion
The test page has two buttons. They are located at 2000 px and 4000 px along the X-axis, respectively (outside of the viewport).
```

**Review by arminal — COMMENTED** [review]
| `y`  | the number of pixels by which the page is scrolled vertically.|
there should be no period here

File: examples/scroll/README.md [hunk 4]
```diff
@@ -2,9 +2,15 @@
...
+>
+> * [t.scroll](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/scroll.html)
+> * [t.scrollBy](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/scrollby.html)
+> * [t.scrollIntoView](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/scrollintoview.html)
+
 TestCafe [actions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html) have a built-in scrolling mechanism. In rare cases, you may need to scroll the page manually. This example demonstrates two common ways to do this: a `hover` action and a [ClientFunction](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html)
 
-The test page has two buttons. They are located at 2000 px and 4000 px along the X-axis, respectively (both outside of the viewport).
+The test page has two buttons. They are located at 2000 px and 4000 px along the X-axis, respectively (outside of the viewport).
```
Comment (arminal) [inline examples/scroll/README.md:13]:
```suggestion
The test page has two buttons. They are located at 2000 px and 4000 px along the X-axis (outside of the viewport), respectively.
```

**Pull request merged**